app/services/resync_service.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+class ResyncService < ApplicationService
+  attr_reader :document
+
+  def initialize(document)
+    @document = document
+  end
+
+  def call
+    update_live_edition(document.live_edition) if document.live_edition
+    update_current_edition(document.current_edition) unless
+      document.current_edition == document.live_edition
```

Comment:
I think these method names could be re-considered to capture what they do - as I find the use of `update` a bit confusing. You might be able to have `sync_live_edition` and `sync_draft_edition`

Reply:
I was going to rename them later when they're actually syncing with publishing API 👍 think I forgot to do that. In the early commits all they do is update the local models.